### ISSUE TITLE
Good to add meta directive with viewport in .rst files in astropy/docs folder.

### ISSUE THREAD

**Issue by webbdays** [astropy/astropy#12358]
meta directive with viewport in .rst files.

.. meta::
   :viewport: initial_scale=1, width=device - width

By adding this meta directive with viewport
makes the docs website mobile friendly for viewing.

**Comment by abhro** [comment]
Sphinx does this by default now, so this can probably be closed.

**Comment by pllim** [comment]
Yeah actually we have switched theme a couple of times now. Should be mobile friendly. Thanks for the ping!
